cartridges/bm_algolia/cartridge/templates/resources/algolia.properties

```
@@ -12,11 +12,22 @@ algolia.label.preference.adminkey=Admin API key
 algolia.label.preference.instock=InStock Threshold
 algolia.label.preference.custom=Additional Product Attributes
 algolia.label.preference.indexprefix=Index Prefix. If set, it replaces the default index prefix:
-algolia.label.preference.indexprefix.example=<hostname>__<siteID>
-algolia.label.preference.enablessr=Enable server-side rendering for CLP (category landing page) results. Improves SEO.
+algolia.label.preference.enablessr=Enable server-side rendering
 algolia.label.preference.clientid=OCAPI Client ID
 algolia.label.preference.clientpassword=OCAPI Client password
 algolia.label.button.apply=Apply
+algolia.label.util.optional=(Optional)
+algolia.label.util.required=*
+
+# Preferences Info
+algolia.label.preference.enable.help=Enable or disable Algolia indexing.
```

Actually the v2 indexing jobs don't take into account this setting anymore. It's only used to enable or disable the frontend for SFRA or SG implementations.
```suggestion
algolia.label.preference.enable.help=Enable or disable Algolia on your Storefront.
```